docs/project-ops.md

```diff
@@ -0,0 +1,377 @@
+    openidconnectpresets.settings.gardener.cloud    []                       []                 [create delete deletecollection get list patch update watch]
+    cloudprofiles.core.gardener.cloud               []                       []                 [get list watch]
+    projects.core.gardener.cloud                    []                       [flowering]             [get patch update delete]
+    namespaces                                      []                       [garden-flowering]      [get]
+    ```
+
+2. Try to execute an operation that you aren’t allowed, for example: 
+
+    ```
+    kubectl get projects
+    ```
+
+    You receive an error message like this:
+
+    ```
+    Error from server (Forbidden): projects.core.gardener.cloud is forbidden: User "system:serviceaccount:garden-flowering:robot" cannot list resource "projects" in API group "core.gardener.cloud" at the cluster scope
+    ```
+
+## Working with projects
+
+1. You can get the details for a project, where you’re a member (or through a service account). 
+
+    ```
+    kubectl get project flowering
+    ```
+
+    The response looks like this:
+
+    ```
+    NAME        NAMESPACE          STATUS   OWNER                    CREATOR                         AGE
+    flowering   garden-flowering   Ready    [PROJECT-ADMIN]@domain   [PROJECT-ADMIN]@domain system   45m
+    ```
+
+    > For more information, see [Project](https://gardener.cloud/api-reference/core/#core.gardener.cloud/v1beta1.Project) in the API reference.
+
+2. To query the names of the members of a project, use the following command:
+   
+    ```
+    kubectl get project docu -o jsonpath='{.spec.members[*].name }'
+    ```
+
+    The response looks like this:
+
+    ```
+    [PROJECT-ADMIN]@domain system:serviceaccount:garden-flowering:robot
+    ```
+
+    > For more information, see [members](https://gardener.cloud/api-reference/core/#core.gardener.cloud/v1beta1.ProjectMember) in the API reference.
+
+## Working with clusters
+
+The Gardener domain object for a managed cluster is called [Shoot](https://gardener.cloud/api-reference/core/#core.gardener.cloud/v1beta1.Shoot).  
+
+### List project clusters
+
+To query the clusters in a project:
+```
+kubectl get shoots
+```
+
+The output looks like this:
+
+```
+NAME       CLOUDPROFILE   VERSION   SEED      DOMAIN                                 HIBERNATION   OPERATION   PROGRESS   APISERVER   CONTROL   NODES   SYSTEM   AGE
+geranium   aws            1.18.3    aws-eu1   geranium.flowering.shoot.<truncated>   Awake         Succeeded   100        True        True      True    True     74m
+```
+### Create a new cluster
+
+To create a new cluster using the command line, you need a YAML definition of the `Shoot` resource. 
+
+1. To get started, copy the following YAML definition to a new file, for example, `daffodil.yaml` (or copy file [shoot.yaml](shoot.yaml) to `daffodil.yaml`).
+
```

Comment:
This should yaml will not work for anyone and I don't like the idea of people opening issues for this. Again, this has nothing to do with the dashboard, this is Gardener API Usage.